src/retasc/validator/models.py

```diff
@@ -0,0 +1,41 @@
+from pydantic import BaseModel, Field
+
+from retasc.validator.config import SCHEMA_VERSION
```

Comment:
It is probably OK to have `SCHEMA_VERSION` defined in this submodule. It is not an "option", but a constant that we would need to bump whenever the models change in non-backwards compatible way.

Reply:
Changed, thanks